src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -6,25 +6,32 @@
     [react-native.safe-area :as safe-area]
     [reagent.core :as reagent]
     [status-im.common.controlled-input.utils :as controlled-input]
+    [status-im.constants :as constants]
     [status-im.contexts.wallet.common.account-switcher.view :as account-switcher]
     [status-im.contexts.wallet.common.asset-list.view :as asset-list]
     [status-im.contexts.wallet.common.utils :as utils]
     [status-im.contexts.wallet.send.input-amount.style :as style]
     [status-im.contexts.wallet.send.routes.view :as routes]
     [status-im.contexts.wallet.send.utils :as send-utils]
     [utils.address :as address]
+    [utils.debounce :as debounce]
     [utils.i18n :as i18n]
     [utils.money :as money]
+    [utils.number :as number]
     [utils.re-frame :as rf]))
 
-(defn- make-limit-label
+(defn- make-limit-label-crypto
   [amount currency]
   (str amount
        " "
        (some-> currency
                name
                string/upper-case)))
 
+(defn- make-limit-label-fiat
+  [amount currency-symbol]
+  (str currency-symbol amount))
```

Comment:
Formatting for fiat was incorrect.

Previously it showed like: `100 EUR`, now it should show like `€100`

Reply:
I think that depends on the language/region?

I've seen in some places in Europe they use `100 €` and in USA and Mexico we use `€ 100`.

So we might need to move this to translations